src/libblastrampoline.c

```diff
@@ -401,7 +401,7 @@ LBT_DLLEXPORT int32_t lbt_forward(const char * libname, int32_t clear, int32_t v
  */
 #ifdef _OS_WINDOWS_
 void * _win32_self_handle;
-BOOL APIENTRY DllMain(HINSTANCE hModule, DWORD code, void *) {
+BOOL APIENTRY DllMain(HINSTANCE hModule, DWORD code, void *reserved) {
```

Comment:
Maybe `unused` is a more accurate description of the variable?
```suggestion
BOOL APIENTRY DllMain(HINSTANCE hModule, DWORD code, void *unused) {
```

Reply:
Windows docs call it reserved: https://learn.microsoft.com/en-us/windows/win32/dlls/dllmain. I think it is better to use the same here.